generated/vetted.tokenlist.json

```
@@ -4077,20 +4089,310 @@
+        },
+        {
+            "address": "0x758B4684BE769E92eeFeA93f60DDA0181eA303Ec",
+            "chainId": 1,
+            "name": "Phonon DAO",
+            "symbol": "PHONON",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0x758b4684be769e92eefea93f60dda0181ea303ec.png"
+        },
+        {
+            "address": "0x0C10bF8FcB7Bf5412187A595ab97a3609160b5c6",
+            "chainId": 1,
+            "name": "Decentralized USD",
+            "symbol": "USDD",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0x0c10bf8fcb7bf5412187a595ab97a3609160b5c6.png"
+        },
+        {
+            "address": "0x853d955aCEf822Db058eb8505911ED77F175b99e",
+            "chainId": 1,
+            "name": "Frax",
+            "symbol": "FRAX",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x853d955aCEf822Db058eb8505911ED77F175b99e/logo.png"
+        },
+        {
+            "address": "0x8888801aF4d980682e47f1A9036e589479e835C5",
+            "chainId": 1,
+            "name": "88mph.app",
+            "symbol": "MPH",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0x8888801af4d980682e47f1a9036e589479e835c5.png"
+        },
+        {
+            "address": "0x470EBf5f030Ed85Fc1ed4C2d36B9DD02e77CF1b7",
+            "chainId": 1,
+            "name": "Temple",
+            "symbol": "TEMPLE",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0x470ebf5f030ed85fc1ed4c2d36b9dd02e77cf1b7.png"
+        },
+        {
+            "address": "0xd084944d3c05CD115C09d072B9F44bA3E0E45921",
+            "chainId": 1,
+            "name": "Manifold Finance",
+            "symbol": "FOLD",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0xd084944d3c05cd115c09d072b9f44ba3e0e45921.png"
+        },
+        {
+            "address": "0xc55126051B22eBb829D00368f4B12Bde432de5Da",
+            "chainId": 1,
+            "name": "BTRFLY",
+            "symbol": "BTRFLY",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0xc55126051b22ebb829d00368f4b12bde432de5da.png"
+<<<<<<< HEAD
+        },
+        {
+            "address": "0x81f8f0bb1cB2A06649E51913A151F0E7Ef6FA321",
+            "chainId": 1,
+            "name": "VitaDAO Token",
+            "symbol": "VITA",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0x81f8f0bb1cb2a06649e51913a151f0e7ef6fa321.png"
+=======
+>>>>>>> a366b2d45c09b3209350deead9b0ab690e95df54
```

Same again here, you need to pick which version you want, which in this case looks like the addition of Vita or not.
```suggestion
        },
        {
            "address": "0x81f8f0bb1cB2A06649E51913A151F0E7Ef6FA321",
            "chainId": 1,
            "name": "VitaDAO Token",
            "symbol": "VITA",
            "decimals": 18,
            "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0x81f8f0bb1cb2a06649e51913a151f0e7ef6fa321.png"
```